### PR TITLE
Dockerize app - Dockerfile and changes in application.properties

### PR DESCRIPTION
**Pull request by krishnakrmahto97**

### CODE REVIEW

**Comment by prashantkumar1997** [comment]
Can we add separate profiles for development and deployment

**Comment by mroyme** [comment]
> Can we add separate profiles for development and deployment

We can, but do we have anything right now to differentiate production from development?
I mean those environment variables can be easily passed in local environment as well.

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/inaam
-spring.datasource.username=postgres
-spring.datasource.password=postgres
+spring.datasource.url=jdbc:postgresql://${SPRING_DATASOURCE_HOST}:${SPRING_DATASOURCE_PORT}/${SPRING_DATASOURCE_DATABASE}
```
Comment (mroyme) [inline src/main/resources/application.properties:1]: Allow usage of other DB drivers.

```suggestion
spring.datasource.url=jdbc:${SPRING_DATASOURCE_DRIVER}://${SPRING_DATASOURCE_HOST}:${SPRING_DATASOURCE_PORT}/${SPRING_DATASOURCE_DATABASE}
```

**Review by mroyme — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (dhawalnegi1) [reply src/main/resources/application.properties]: Added usage of other DB drivers

**Pull request merged**